src/qt/forms/optionsdialog.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>560</width>
-    <height>440</height>
+    <width>646</width>
```

Comment:
Nit

 there's not really a reason to change the size here. I guess this is a good place to answer a question:

"When adding new settings, should we maintain a certain level of padding between the last setting option and its enclosing box?"

| Use PR Size | Use Master Size |
| ----------------- | ----------------------- |
| ![size-increase](https://user-images.githubusercontent.com/23396902/120907709-bbb79580-c631-11eb-8755-6d7bac8f0f76.png) | ![size-normal](https://user-images.githubusercontent.com/23396902/120907710-bf4b1c80-c631-11eb-85b8-f4a7d2f3cca2.png) |

Reply:
> there's not really a reason to change the size here. I guess this is a good place to answer a question:

The reason is that successive openings and closings of the `optionsdialog.ui` in Qt Designer won't suggest size changes.